clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/handler/KubernetesHandler.java

```diff
@@ -45,10 +45,20 @@
 
   private final ArtifactReplacer artifactReplacer;
 
+  protected String dockerImageBinding;
```

Comment:
Have you tried to inject the value here with `@Value` instead of injecting in every subclass? In theory it should work

Reply:
Yeah, you are right, it works, I did some test with this in my first approach but didn't work, never tried that in this approach.